pepper-apis/build-utils/build-support-images/Dockerfile-build

```diff
@@ -0,0 +1,67 @@
+# To make our config generation "compatible" with official dsdetoolbox, extract what is needed from it (this is not the base image)
+FROM broadinstitute/dsde-toolbox:DDO-1934 AS dsdetoolbox
+
+FROM maven:3.6.3-jdk-11 AS maven
+
+FROM google/cloud-sdk:alpine
+
+#Essentials for Maven
+COPY --from=maven /usr/share/maven /usr/share/maven
+RUN ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+COPY --from=maven /usr/local/bin/mvn-entrypoint.sh /usr/local/bin/mvn-entrypoint.sh
+COPY --from=maven /usr/share/maven/ref/settings-docker.xml /usr/share/maven/ref/settings-docker.xml
+
+ENV MAVEN_HOME /usr/share/maven
+ENV MAVEN_CONFIG "$USER_HOME_DIR/.m2"
+
+# Copying just configuration used by official DSDE
+COPY --from=dsdetoolbox /etc/consul-template/config/config.json /etc/consul-template/config/config.json
+ENV VAULT_ADDR https://clotho.broadinstitute.org:8200
+ENV CONSUL_TEMPLATE_VERSION=0.27.2
+ENV VAULT_VERSION=1.4.3
+
+# Set by build
+ARG TARGETARCH
+
+RUN case "${TARGETARCH:-amd64}" in \
+           arm64) export HASHICORP_ARCH='arm64' ;; \
+           amd64) export HASHICORP_ARCH='amd64' ;; \
+    esac; \
+    apk  --no-cache add \
+    curl \
+    bash \
+    jq \
+    mysql-client \
+    netcat-openbsd \
+    git \
+    openjdk11 \
+    # to support configure.rb
+    ruby \
+    ruby-json  \
+    # to support sheetappend script & OpenAPI build/deploy
+    nodejs \
+    npm \
+    && rm -rf /var/cache/apk/* \
+    # Get the architecture specific builds of vault and consul template
+    && wget https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_${HASHICORP_ARCH}.zip -O /tmp/vault_${VAULT_VERSION}_linux.zip \
+    && wget --no-check-certificate https://releases.hashicorp.com/consul-template/${CONSUL_TEMPLATE_VERSION}/consul-template_${CONSUL_TEMPLATE_VERSION}_linux_${HASHICORP_ARCH}.zip -O /tmp/consul_template_${CONSUL_TEMPLATE_VERSION}_linux.zip \
+    && cd /tmp  \
+    && unzip vault_${VAULT_VERSION}_linux.zip  \
+    && unzip consul_template_${CONSUL_TEMPLATE_VERSION}_linux.zip  \
+    && mv consul-template vault /usr/local/bin/  \
+    && chmod 0755 /usr/local/bin/consul-template \
+    && chmod 0755 /usr/local/bin/vault \
+    && rm -rf /tmp/*.zip
```

Comment:
Got excited to see these in the repo and tried it. Alas, vault installation seems to be broken/vault does not execute locally on my M1 or in presumably amd/64 on CircleCI. And consul-template just hangs.
Going to guess that this is probably why DevOps was wget'ting builds rather than using apk: they ran into the same problem I just did.
Going with what work here.